packages/rum/src/typesV2.ts

```diff
@@ -148,3 +148,15 @@ export type RumEventV2 =
   | RumViewEventV2 & ViewContextV2 & RumContextV2
   | RumLongTaskEventV2 & ActionContextV2 & ViewContextV2 & RumContextV2
   | RumActionEventV2 & ViewContextV2 & RumContextV2
+
+export interface User {
+  id?: number | string
+  email?: string
+  name?: string
+  [key: string]: unknown
+}
+
+export interface GlobalAttributes {
```

Comment:
naming: attributes seems inconsistent with context that we are using everywhere. To avoid collision with globalContext we could use something like customerContext or customerCommonContext.
wdyt?

Reply:
What's bothering me with using `context` in the name is that it already contains a `context` name, so we would end up with `customerContext.context`, which is not ideal. Also, it may not be only customer-related data in the future. Maybe we can find something else, like `Environment`?

Reply:
> Maybe we can find something else, like Environment?

IMO we should have a similar naming for this concept and the existing *Context concepts, because the purpose is the same, only the content varies.
I'd prefer having `Context.context` than thinking of another name for this generic concept and make the update everywhere.

> Also, it may not be only customer-related data in the future
we could rename it at this point then or use only commonContext

Reply:
Let's use `CommonContext`